src/coreclr/System.Private.CoreLib/src/System/Reflection/Metadata/AssemblyExtensions.cs

```diff
@@ -12,13 +12,19 @@ public static class AssemblyExtensions
         [return: MarshalAs(UnmanagedType.Bool)]
         private static extern unsafe bool InternalTryGetRawMetadata(QCallAssembly assembly, ref byte* blob, ref int length);
 
-        // Retrieves the metadata section of the assembly, for use with System.Reflection.Metadata.MetadataReader.
-        //   - Returns false upon failure. Metadata might not be available for some assemblies, such as AssemblyBuilder, .NET
-        //     native images, etc.
-        //   - Callers should not write to the metadata blob
-        //   - The metadata blob pointer will remain valid as long as the AssemblyLoadContext with which the assembly is
-        //     associated, is alive. The caller is responsible for keeping the assembly object alive while accessing the
-        //     metadata blob.
+        /// <summary>
+        /// Retrieves the metadata section of the assembly, for use with <see cref="MetadataReader" />.
+        /// </summary>
+        /// <returns>
+        /// Returns <see langword="false" /> upon failure. Metadata might not be available for some assemblies,
+        /// such as <see cref="System.Reflection.Emit.AssemblyBuilder" />, AOT images, etc.
+        /// </returns>
+        /// <remarks>
+        /// <para>Callers should not write to the metadata blob.</para>
+        /// <para>The metadata blob pointer will remain valid as long as the <see cref="System.Runtime.Loader.AssemblyLoadContext" />
+        ///   with which the assembly is associated, is alive.
+        ///   The caller is responsible for keeping the assembly object alive while accessing the metadata blob.</para>
```

Comment:
I'm a bit confused about this part: does the ALC need to be kept alive, or the `Assembly`?

Reply:
Both the Assembly and the owning ALC have same lifetime. Keeping one of them alive will keep the other one alive too.

Reply:
@jkotas Thanks. In that case, I think there is no reason to mention the ALC in the first place. I have updated the comment.